liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -1783,6 +1909,12 @@ public boolean isWindows() {
         private static final String CHANGELOG_FILE = "changeLogFile";
         private static final String DATA_OUTPUT_DIRECTORY = "dataOutputDirectory";
         private static final String DIFF_TYPES = "diffTypes";
+        private static final String CHANGE_SET_ID = "changeSetId";
+        private static final String CHANGE_SET_AUTHOR = "changeSetAuthor";
+        private static final String CHANGE_SET_PATH = "changeSetPath";
+        private static final String OUTPUT_FILE = "outputFile";
```

Comment:
outputFile is a global parameter and so already handled with the outputFile property. It should not be added to the command-level options